### ISSUE TITLE
[Feat]: Support for Audiobooks

### ISSUE THREAD

**Issue by Pfuenzle** [henniaufmrenni/boum#5]
Thanks for you app, stumbled upon it on reddit today and it works really great for music :)

I also have a lot of audiobooks in my library, which currently do not show up.
Have you planned to add support for Audiobooks?
If not, totally fine
Thanks in advance :)

**Comment by wojtek1171** [comment]
+1, and playback speed adjustment would be great

**Comment by henniaufmrenni** [comment]
I'll look into implementing audiobook support, however the book libraries in Jellyfin are folder based and it seems the API doesn't provide a way to get a flat list with all audiobooks. I tried using the [`/Users/{userId}/Items`](https://api.jellyfin.org/#tag/Items/operation/GetItemsByUserId) endpoint with `?includeItemTypes=AudioBook` which returns nothing as it seems that audiobooks in the book libraries are of `"Type": "MusicAlbum"`.
tl;dr: It'll take some hacking to get it working.
